BUILD.bazel

```diff
@@ -286,477 +286,201 @@ cc_library(
 )
```

Comment:
Can you split the lexicographic sorting part of the diff into a separate PR? I'll skip reviewing the build changes in the meanwhile.